### PR TITLE
MEV Capital wETH Cluster

### PR DESCRIPTION
**Pull request by hoytech**

### CODE REVIEW

File: logo/mev-capital.png
Comment (kasperpawlowski) [inline logo/mev-capital.png:1]: that png will have a grid visible, no?

Comment (hoytech) [reply logo/mev-capital.png:1]: I don't think so. It's a PNG with a transparent background, so it depends on what background colour the dapp uses.

File: verify.js [hunk 1]
```diff
@@ -18,7 +18,11 @@ for (const file of Object.keys(logos)) {
 
 	if (info.type !== "svg") {
 		// legacy PNG files: please use SVG instead
-		if (file !== "re7labs.png" && file !== "apostro.png")
+		if (
+			file !== "re7labs.png" &&
```
Comment (conwayconstar) [inline verify.js:22]: Maybe we should allow pngs 😅

**Review by conwayconstar — DISMISSED** [review]

File: logo/mev-capital.png
Comment (conwayconstar) [reply logo/mev-capital.png:1]: We use a dark one so this will not show. Also needs a bit of a bleed

**Review by conwayconstar — APPROVED** [review]

**Review by rori4 — APPROVED** [review]

**Pull request merged**